src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx

```diff
@@ -574,17 +596,106 @@ These Support Plans apply only to your paid service subscription under an existi
           </td>
         </tr>
       </tbody>
-    </table>
+</table>
+
+### Response times 
+
+The table below shows estimated response times based on your New Relic usage plan. 
+
+<table>
+    <thead>
+        <tr>
+            <th>
+                Priority
+            </th>
+            <th>
+                Priority 
+            </th>
+            <th>
+                Platinum 
+            </th>
+            <th>
+                Gold 
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                P1
+            </td>
+            <td>
+                1 hour 
+            </td>
+            <td>
+                1 hour 
+            </td>
+            <td>
+                2 hours 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                P2
+            </td>
+            <td>
+                2 business hours
+            </td>
+            <td>
+                2 business hours
+            </td>
+            <td>
+                6 business hours 
+            </td>
+        </tr>
+        <tr>
+            <td>
+                P3
+            </td>
+            <td>
+                1 business day
+            </td>
+            <td>
+                1 business day
+            </td>
+            <td>
+                1 business days
+            </td>
+        </tr>
+        <tr>
+            <td>
+                P4
+            </td>
+            <td>
+                2 business days
+            </td>
+            <td>
+                2 business days
+            </td>
+            <td>
+                2 business days
+            </td>
+        </tr>
+    </tbody>
+</table>
 
     **Notes:**
 
     * Silver tier applies to customers with $1 to $9,999 annual spend. Gold tier applies to customers with $10,000 to $99,999 annual spend. Platinum tier applies to customers with $100,000 annual spend and above. Contact your Account Manger regarding Priority Support.
-    * Initial Support Response Time begins when the request is received by the New Relic support system.
-    * Critical means customer's business operations are severely impacted due to New Relic with no available workaround; or there is a critical security issue.
-    * This Support Plan is subject to change at any time; changes will take immediate effect.
+    * 24x7x365 support, otherwise known as "Follow the Sun" support, is only applicable to P1 business critical support cases.
+    * Initial support response time begins when the request is received by the New Relic support system.
+    * This support plan is subject to change at any time; changes will take immediate effect.
```

Comment:
@homelessbirds I'm not seeing this definition about critical in this revision. Is that by design? Here's the statement: "Critical means customer's business operations are severely impacted due to New Relic with no available workaround; or there is a critical security issue.

Reply:
This was removed on purpose. Thanks for flagging!